expat/fuzz/xml_parse_fuzzer.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "expat.h"
+#include <assert.h>
+#include <stdint.h>
+
+static void XMLCALL start(void *userData, const char *name, const char **atts) {
+}
+static void XMLCALL end(void *userData, const char *name) {}
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  char *encoders[7] = {"UTF-16",   "UTF-8",    "ISO_8859_1", "US_ASCII",
+                       "UTF_16BE", "UTF_16LE", NULL};
```

Comment:
Pull request #292 has one binary per encoding, we have one binary looping over all. @hctim can you share your thought's why the single binary approach is preferably? @yevgenypats would this approach work for fuzzit?